ledger-core/insolar/payload/localtypes.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package payload
+
+import (
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/rms"
+)
+
+// aliases for more convenient using in .proto file (no need to use full path)
+// because alias in the same package
+type MessageContext = rms.MessageContext
+
+func RegisterMessageType(id uint64, special string, t proto.Message) {
+	rms.RegisterMessageType(id, special, t)
+}
+
+// Payload represents any kind of data that can be encoded in consistent manner.
+type Payload interface {
+	Marshal() ([]byte, error)
+}
+
+type Reference = reference.Global
+type LocalReference = reference.Local
+
+type PulseNumber = pulse.Number
+
+////go:generate protoc -I=. -I=E:\Go\src --ins_out=./ proto_*.proto
```

Comment:
-I=E:\Go\src ?
remove please

Reply:
All that comment could be deleted, it won't work as go gen subcommand, as i remember

Reply:
This is mine for local generation. Sorry